components/api/src/test/java/com/hotels/styx/api/MockContext.java

```diff
@@ -34,6 +34,7 @@ public void add(String key, Object value) {
     }
 
     // TODO: Mikko: Styx 2.0 API: MockObservable support for `onError`.
+    // This class is never used.
     static class MockObservable<T> implements StyxObservable<T> {
```

Comment:
If the class is never used, might be worth removing the `MockObservable` class too? Or at least add a comment to justify why it is still necessary.

Reply:
I'll remove it if you don't want it. I don't know what it is for.

Reply:
Please do.